files/mender/mender.conf.template

```diff
@@ -0,0 +1,9 @@
+{
+    "InventoryPollIntervalSeconds": 5,
```

Comment:
We'll need parameters to specify poll interval
https://docs.mender.io/1.7/client-configuration/configuration-file/polling-intervals#how-to-choose-right-intervals

Reply:
Or even a set of parameters to provide all possible key-value pairs in mender.conf

Reply:
Just wanted to highlight that we changed these in the mender-convert tool as well, https://github.com/mendersoftware/mender-convert/commit/366cf37453e8728a6c68e622b0428f1ecfcc5fec.

Also wanted to point that that these are "demo" values only intended to be used together with the [Demo environment](https://docs.mender.io/1.6/getting-started/create-a-test-environment).

We default to these values in a "production" build, 

MENDER_UPDATE_POLL_INTERVAL_SECONDS ?= "1800"
MENDER_INVENTORY_POLL_INTERVAL_SECONDS ?= "1800"
MENDER_RETRY_POLL_INTERVAL_SECONDS ?= "300"

Reply:
re #129